metadata-service/restli-client/src/main/java/com/linkedin/entity/client/JavaEntityClient.java

```diff
@@ -374,4 +374,20 @@ public String ingestProposal(@Nonnull MetadataChangeProposal metadataChangePropo
         }
         return Optional.empty();
     }
+
+    @SneakyThrows
+    public DataMap getRawAspect(@Nonnull String urn, @Nonnull String aspect,
+        @Nonnull Long version, @Nonnull Authentication authentication) throws RemoteInvocationException {
+        VersionedAspect entity = _entityService.getVersionedAspect(Urn.createFromString(urn), aspect, version);
+        if (entity == null) {
+            return null;
+        }
+
+        if (entity.hasAspect()) {
```

Comment:
Good stuff! Is there ever a reason we'll want an "Enveloped Aspect" hre? That is the raw aspect PLUS information about when / by who it was created, the version, the system metadata, etc

Reply:
yeah- i agree that endpoint would be useful! i left space in the graphql response to add that in later 👍